webplatform/Initial application page html/learn.html

```diff
@@ -0,0 +1,187 @@
+<!DOCTYPE html>
```

Comment:
Could you move only html to app/views/index.html.slim (that's our home page)? In this file you put only html which is inside <body>, the rest is unnecessary. As for css please create new file in app/assets/stylesheets/application folder for example homepage.scss and then put there all your all css. Next can you use bootstrap for html (http://getbootstrap.com/)? It's easier to make the page responsive using this framework :) And at the end if you could change html to slim - you can use this page https://html2slim.herokuapp.com/ :)